doc/src/manual/ranch.asciidoc

```diff
@@ -107,6 +114,38 @@ options and transport-specific socket options.
 
 None of the options are required.
 
+alarms (#{})::
+
+Alarms to call a function when the number of connections under a
+connection supervisor reaches or exceeds a defined treshold.
++
+The map keys are the alarm names, which can be any `term`. The
+associated values are the respective alarm options, again in a map
+with the following keys:
+
+type (required):::
```

Comment:
Remove `(required)` since it has no defaults it shouldn't list anything otherwise we'll think `required` is the default.

Reply:
Done.